vscode-trace-extension/src/trace-explorer/trace-tree.ts

```diff
@@ -311,8 +311,8 @@ const isCtf = async (directory: string): Promise<boolean> => {
 };
 
 function getProgressBarTitle(traceUri: vscode.Uri | undefined): string {
-    if (!traceUri || !traceUri.path) {
+    if (!traceUri || !traceUri.fsPath) {
         return 'undefined';
     }
-    return traceUri.path.substring(traceUri.path.lastIndexOf('/') + 1);
+    return traceUri.fsPath.substring(traceUri.fsPath.lastIndexOf('/') + 1);
```

Comment:
what this suppose to be? basename? Path should use node path module to get file basename

Reply:
Fixed in latest commit.